tests/legacy-cli/e2e.bzl

```diff
@@ -119,6 +126,23 @@ def _e2e_tests(name, runner, **kwargs):
     toolchains = toolchains + ["@npm//@angular/build-tooling/bazel/browsers/chromium:toolchain_alias"]
     data = data + ["@npm//@angular/build-tooling/bazel/browsers/chromium"]
 
+    windows_node_repo = kwargs.pop("windows_node_repo", None)
```

Comment:
How does all of this work below if `windows_node_repo` is actually `None`?

For instance `"@//:node_files"` is invalid right?